### PR TITLE
Timer Entity

### LINKED ISSUE #3: [Feature 5] Timer
Timer allows the user to work on a particular task for a set amount of time without any distractions. They start a countdown timer on their device to time their task.

User can also choose to take breaks between their work sessions.

### PR DESCRIPTION
**Pull request by paridhi26**
Created the Timer Enitity. This is the almost ready timer entity execpt the part where I need the current userid.  The parts that others need to use this entity is complete, so there should be no problem if others need to use this class in their code.

### CODE REVIEW

**Review by 1StevenJiang1 — DISMISSED** [review]

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
```
Comment (Dawei-He2002) [inline gradle/wrapper/gradle-wrapper.properties]: Please leave the gradle files and stuff on gitignore or not commit them.

**Review by Dawei-He2002 — DISMISSED** [review]
Otherwise looks good.

**Review by 1StevenJiang1 — DISMISSED** [review]

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (paridhi26) [reply gradle/wrapper/gradle-wrapper.properties]: Yes, I added .idea and .graddle to gitignore, so this problem should not occur in the future.

**Review by chazzybearr — APPROVED** [review]

**Review by 1StevenJiang1 — APPROVED** [review]

**Review by nitvishn — APPROVED** [review]
Looks good!

**Pull request merged**